docs/config/README.md

```diff
@@ -69,6 +69,18 @@ export MINIO_BROWSER=off
 minio server /data
 ```
 
+#### TLS
+|Field|Type|Description|
+|:---|:---|:---|
+|``cert_passwd``| _string_ | Password to decrypt the TLS private key. Must be set if the TLS private key is password protected - See [TLS configuration](https://docs.minio.io/docs/how-to-secure-access-to-minio-server-with-tls)|
+
```

Comment:
This entry here only means we are making an entry into config.json which is not the case in this PR. MINIO_CERT_PASSWD is just an env so we should mention it differently.

Reply:
Oh sry - thought all ENV are mentioned here - remove the entry.
Okay so it is mentioned in the TLS related docs - that should be sufficient.